site/docs/components/icons.md

```diff
@@ -114,40 +177,18 @@ The following brand-specific color classes are provided:
 {% endcapture %}
 {% include example.html content=example %}
 
-### Transform
 
-Transform icons with [transform utilities](/docs/utilities/transform)
+## Formatting a new SVG
 
-{% capture example %}
-<svg class="rotate-180">
-  <use xlink:href="#right_arrow" />
-</svg>
-{% endcapture %}
-{% include example.html content=example %}
+Each icon in the system is a formatted svg with a `viewBox="0 0 500 500"`, a path fill="currentColor" and a descriptive class name.
 
-## All Icons
+The SVG is created on a 500px x 500px art board. SVG's created in a design program such as Adobe Illustrator or Sketch contain extra information. Clean up an svg with this online tool https://jakearchibald.github.io/svgomg/. Finally format the svg to match the example below.
+ 
 
 {% capture example %}
-<div class="row">
-{% for icon in site.data.icons %}
-<div class="col-6 col-md-2 d-flex flex-column align-items-center justify-content-center">
-  <svg class="icon">
-    <use xlink:href="#{{ icon.name }}" />
-  </svg>
-  <p class="caption">{{ icon.name }}</p>
-</div>
-{% endfor %}
-</div>
-<div class="row text-white bg-black">
-{% for icon in site.data.icons %}
-<div class="col-6 col-md-2 d-flex flex-column align-items-center justify-content-center">
-  <svg class="icon">
-    <use xlink:href="#{{ icon.name }}" />
-  </svg>
-  <p class="caption">{{ icon.name }}</p>
-</div>
-{% endfor %}
-</div>
 
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 500" class="icon-cog">
+<path fill="currentColor" d="M346.382,250.246A96.882,96.882,0,1,0,249.5,347.128,96.992,96.992,0,0,0,346.382,250.246Zm93.428,35.383,42.96,24.8a12.121,12.121,0,0,1,5.5,14.1,249.617,249.617,0,0,1-55.13,95.348,12.113,12.113,0,0,1-14.952,2.273l-42.927-24.794a193.113,193.113,0,0,1-61.249,35.413v49.579a12.1,12.1,0,0,1-9.453,11.812,251.979,251.979,0,0,1-110.1.008,12.123,12.123,0,0,1-9.479-11.818V432.764a193.083,193.083,0,0,1-61.249-35.413L80.808,422.145a12.113,12.113,0,0,1-14.952-2.273,249.63,249.63,0,0,1-55.13-95.348,12.122,12.122,0,0,1,5.5-14.1l42.961-24.8a195.04,195.04,0,0,1,0-70.765l-42.96-24.8a12.121,12.121,0,0,1-5.5-14.1A249.621,249.621,0,0,1,65.857,80.62a12.113,12.113,0,0,1,14.952-2.272l42.927,24.793a193.131,193.131,0,0,1,61.249-35.413V18.149a12.108,12.108,0,0,1,9.453-11.813,252,252,0,0,1,110.1-.007,12.121,12.121,0,0,1,9.479,11.817V67.727a193.1,193.1,0,0,1,61.249,35.413l42.927-24.793a12.113,12.113,0,0,1,14.952,2.272,249.634,249.634,0,0,1,55.13,95.349,12.122,12.122,0,0,1-5.5,14.1l-42.961,24.8A195.045,195.045,0,0,1,439.81,285.629Z"/>
+</svg>
 {% endcapture %}
 {% include example.html content=example %}
```

Comment:
Can we point out the important parts in bullet points?

- fill is currentColor
- viewBox is present
- no inline width, height, or colors
- use of classes to reference the SVG
- the svg is flat with only a path tag. No g tags, masks, or nested paths